tests/moonbeam-test-specs/templates/simple-specs-template.json

```diff
@@ -34,7 +34,10 @@
           }
         }
       },
-      "palletEthereum": {}
+      "palletEthereum": {},
+      "stake": {
+        "stakers": [["0x6be02d1d3665660d22ff9624b7be0551ee1ac91b", null, 100]]
```

Comment:
the genesis account is the only endowed balance account so it is also the only validator

consequence is that the `test-balance` expected amounts are 100 less than what they were before (because 100 is bonded for the validator)